x-pack/plugins/security_solution/public/timelines/components/side_panel/event_details/flyout/back_to_alert_details_link.tsx

```diff
@@ -5,34 +5,58 @@
  * 2.0.
  */
 
-import { EuiButtonEmpty, EuiText, EuiTitle } from '@elastic/eui';
+import {
+  EuiBetaBadge,
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import { css } from '@emotion/react';
 import React from 'react';
 import {
   ISOLATE_HOST,
   UNISOLATE_HOST,
 } from '../../../../../detections/components/host_isolation/translations';
-import { ALERT_DETAILS } from '../translations';
+import { ALERT_DETAILS, TECHNICAL_PREVIEW, TECHNICAL_PREVIEW_DESCRIPTION } from '../translations';
 
 const BackToAlertDetailsLinkComponent = ({
   showAlertDetails,
+  showExperimentalBadge,
   isolateAction,
 }: {
   showAlertDetails: () => void;
+  showExperimentalBadge?: boolean;
   isolateAction: 'isolateHost' | 'unisolateHost';
-}) => {
-  return (
-    <>
-      <EuiButtonEmpty iconType="arrowLeft" iconSide="left" flush="left" onClick={showAlertDetails}>
-        <EuiText size="xs">
-          <p>{ALERT_DETAILS}</p>
-        </EuiText>
-      </EuiButtonEmpty>
-      <EuiTitle>
-        <h2>{isolateAction === 'isolateHost' ? ISOLATE_HOST : UNISOLATE_HOST}</h2>
-      </EuiTitle>
-    </>
-  );
-};
+}) => (
+  <>
+    <EuiButtonEmpty iconType="arrowLeft" iconSide="left" flush="left" onClick={showAlertDetails}>
```

Comment:
I'm assuming you are keeping this change even after v8.13 (where we will likely revert this change, since its 8.12 only), correct?  if so, then would suggest adding tests for this change, but I'll defer to the alerts/detections team.